system-tests/services/_scripts/setup-external-volumes

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# create simple app
+cat <<EOF | dcos marathon app add
```

Comment:
When running in verbose mode, the driver logs the STDOUT/STDERR in the console. I think we are already running the job in verbose mode in Jenkins.

Look for `Running lifecycle script "setup"` and the `E:` (stderr) or `O:` (stdout) lines.